dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -3241,98 +3461,205 @@ def snapshot_page(n_clicks, url, wait_selector):
         ```
-        ''',
-        style=styles.code_container),
+        '''
+    ),
 
-        rc.Markdown(
+    rc.Markdown(
         '''
         &nbsp;
 
         If you're receiving the above user permission error, please
         <dccLink href="/dash-enterprise/support" children="contact support"/>.
-        ''')
-    ]),
+        '''
+    ),
 
-    html.Details([
-        html.Summary("Unable to select a buildpack"),
+    rc.Markdown(
+        '''
+        ### fatal: Authentication failed for 'https://<your-dash-enterprise-host>/GIT/<your-dash-app>/'
+        '''
+    ),
 
-        rc.Markdown(
-            '''
-            ```shell
-            ...
-            remote:            Adding BUILD_ENV to build environment...
-            remote:            Unable to select a buildpack
-            ```
-            ''',
-                              style=styles.code_container),
-        rc.Markdown(
-            '''
-            &nbsp;
+    rc.Markdown(
+        '''        
+        ```shell
+        $ git push plotly master
 
-            This error might occur if you are trying to push from a branch
-            that is not your `master` branch. Get the name of your current
-            branch by running `git branch`. Then, to push from this branch
-            to the remote server, run `git push plotly your-branch-name:master`.
+        ...
 
-            &nbsp;
-            '''
-        ),
-    ]),
+        fatal: Authentication failed for 'https://<your-dash-enterprise-host>/GIT/<your-dash-app>/'
+        ```
+        '''
+    ),
 
-    rc.Markdown('''
-    ***
+    rc.Markdown(    
+        '''
+        This will happen when using git to push changes to a SAML enabled Dash Enterprise Server with `https`. Resolve this by <dccLink href="/dash-enterprise/ssh" children="deploying with SSH"/>.
+        '''
+    ),
 
-    #### Problems Using a Celery Process?
 
-    '''),
+    rc.Markdown(
+        '''
+        ### Collabortive deploys
+        '''
+    ),
 
-    html.Details([
-        html.Summary("Callbacks using async processes aren't running and `Celery` is not present in app logs"),
+    rc.Markdown(
+        '''
+        ```shell
+        $ git push plotly master
+        [...]
+        User does not have permissions to run git-receive-pack on <repository>, or <repository> does not exist
+        Access denied
+        fatal: Could not read from remote repository.
+
+        Please make sure you have the correct access rights
+        and the repository exists.
+        ```        
+        '''
+    ),
 
-        html.Br(),
+    rc.Markdown(
+        '''
+        This happens when multiple user push changes to the same app.\
+        
+        Dash Enterprise does not support collaborative deploys or changes being pushed to an app by multiple users. 
+        Only the app owner and admin users can deploy to the application.\
```

Comment:
```suggestion
         You have three options to work deploy apps collaboratively:
```

🔪 since we enumerate the workarounds below.